.github/workflows/build.yml

```
@@ -47,6 +47,10 @@ jobs:
         if: ${{ matrix.api-version == 6.5 }}
         run: src/tools/generate_sysroot.py --api-version 6.5 --out src/sysroot-6.5
 
+      - name: Generate Tizen 8.0 sysroot
+        if: ${{ matrix.api-version == 8 }}
```

```suggestion
        if: ${{ matrix.api-version == 8.0 }}
```
Note that the official name is always "Tizen 8.0" but not "Tizen 8".

---

https://github.com/flutter-tizen/embedder/pull/62 I modified this patch to recognize the version as a string.